src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/Expense.java

```diff
@@ -0,0 +1,28 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import com.pmm.budgetcalculator.foundation.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Entity;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@AllArgsConstructor
+
+public class Expense extends BaseEntity {
+    long expenseId;
+    String expenseName;
+    String expenseDescription;
+    LocalDateTime expenseTime;
+    double expenseAmount; //or BigDecimal
+    String expensePlace;
+
+    public Expense(){
+        this.expenseTime = LocalDateTime.now();
```

Comment:
What if I forgot to add expense yesterday and want to add it today? This date will always be pointing at the moment I'm adding expense in application.
If you want to have some kind of audit when given record was saved and modified read about @CreatedDate and @LastModifiedDate annotations.

Reply:
how about this pattern :     public Expense(String expenseTime) {
this.expenseTime = "dd-MM-yyyy";
} 
in th other hand I'm thinking about making some kind of Calendar and possible to manually choose data, but idk how to implement

about audit, I made in BaseEntity:
@EntityListeners(AuditingEntityListener.class)

@CreatedDate
@Column(name = "registration_time")
protected Date registrationTime;
and I have a problem with POSTMAN, I see null all the time, do you know what am I doing wrong?